vacationtime-solution/SampleData.csv

```diff
@@ -0,0 +1,5 @@
+sholmes@email.com, Sherlock Holmes, manager@gmail.com, June 20, June 25, Vacation with family
```

Comment:
Make this .xslx like: 
https://github.com/gsuitedevs/solutions/tree/master/equipment-requests/data